### ISSUE TITLE
Download, Compile, and Run GHGF With Benchmark Case

### ISSUE THREAD

**Issue by ddalton-swe** [JPL-MGHG/JPL-MGHG-SDS#8]

**Comment by ddalton-swe** [comment]
@bkabyrne Is GHGF this (https://github.com/bkabyrne/CMS_Flux_model.git)?

**Comment by bkabyrne** [comment]
Yes. In the past GHGF and CMS-Flux have been used interchangeably. That is the code that I demonstrated this morning

**Comment by ddalton-swe** [comment]
Got it! Could you provide a run guide as documentation? mostly so the layperson can see what commands are needed to run this.

**Comment by bkabyrne** [comment]
Yes, I will create that as part of the flow chart.

**Comment by ddalton-swe** [comment]
@bkabyrne has provided the flowchart in Slack. [GHGF_flowchart_v0.pptx](https://github.com/JPL-MGHG/JPL-MGHG-SDS/files/13565032/GHGF_flowchart_v0.pptx)

**Comment by ddalton-swe** [comment]
@bkabyrne will provide a benchmark case for GHGF (BLOCKER)

**Comment by gchang** [comment]
@vineety needs access to @bkabyrne no backup directories in Pleiades. Per @ddalton-jpl no longer a blocker.

**Comment by bkabyrne** [comment]
I think access is needed for /nobackupp17/bbyrne1/CMS_Flux_model/Datasets. I have just changed the permission, please try now
